config/samples/bmc_v1alpha1_bmcjob.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: bmc.tinkerbell.org/v1alpha1
+kind: BMCJob
+metadata:
+  name: bmcjob-sample
+spec:
+  tasks:
+    - powerAction:
+        powerControl: "off"
+    - oneTimeBootDeviceAction:
+        device: "pxe"
```

Can you update this to be that slice?
```suggestion
        devices: ["pxe"]
```